### DISCUSSION TITLE
How rtrigger AlertDialog component without button

### DISCUSSION THREAD

**Discussion by uriya4647** [adobe/react-spectrum#5893] [General]
How can rtrigger the AlertDialog without the button.
I want to pop up warning just after i will get the answer from the server.

**Comment by reidbarber** [comment]
You can conditionally render it inside a [DialogContainer](https://react-spectrum.adobe.com/react-spectrum/DialogContainer.html). Check out the first example on that page.

**Comment by uriya4647** [comment]
Thanks, it helped me a lot.
